docs/index.md

```diff
@@ -66,6 +66,7 @@ On non-AMD platforms, like NVIDIA, HIP provides header files required to support
 * [HIP examples](https://github.com/ROCm/HIP-Examples)
```

Comment:
Please consider adding [https://github.com/ROCm/rocm-examples] to link to example and add a more direct link to where users my find the code in this tutorial.

Reply:
I will do this change on a separate PR.

Reply:
https://github.com/ROCm/HIP/pull/3540